src/main/java/net/neoforged/neoforge/common/conditions/NeoForgeConditions.java

```diff
@@ -36,28 +36,28 @@ static ICondition or(ICondition... values) {
         return new OrCondition(List.of(values));
     }
 
-    static <TRegistry> ICondition elementExists(ResourceKey<TRegistry> registryKey) {
+    static <TRegistry> ICondition isRegistered(ResourceKey<TRegistry> registryKey) {
```

Comment:
I would prefer just `registered`. That's also more consistent with `itemRegistered`.

Reply:
yeah tbh i couldnt decide which to use, went with `isRegistered` as it reads better imo, but like you say `registered` does match more with the other names.